src/main/java/core/basesyntax/service/UserService.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.Operation;
+import core.basesyntax.strategy.OperationService;
+import java.util.Map;
+
+public interface UserService {
+    void formReport(Map<Operation, OperationService> operationMap, String fromFile, String toFile);
```

Comment:
`String fromFile` - you need to use your db to create a report.
`Map<Operation, OperationService> operationMap` - it's strategy pattern.

Reply:
This service did everything from start to finish, so I had to send both the input file and the map there, but I understand what the mistake was, so thank you